pages/Footer.js

```diff
@@ -0,0 +1,58 @@
+import 'react-native-gesture-handler';
+import React from 'react';
+import { View, Linking, StyleSheet } from 'react-native';
+import FooterButton from '../components/FooterButton.js';
+
+
+const styles = StyleSheet.create({
+    styledFooter: {
+        height: '100%',
+        width: '100%',
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'space-between',
+    },
+
+});
+
+export const Footer = ({ navigation }) => {
+    return (
+
+        <View style={styles.styledFooter}>
+            <FooterButton
+                onPress={() => navigation.navigate('About')}
+                source={require("../assets/genconnect_footerbuttons_about.png")}
+            />
+            <FooterButton
+                source={require("../assets/genconnect_footerbuttons_share.png")}
+                onPress={() =>
+                    Linking.openURL(
+                        'https://apps.apple.com/us/app/gen-connect/id1588977864'
+                    )
+                }
+            />
+            <FooterButton
+                source={require("../assets/genconnect_footerbuttons_review.png")}
+                onPress={() =>
+                    Linking.openURL(
+                        'https://apps.apple.com/us/app/gen-connect/id1588977864'
```

Comment:
this link is invalid when opened on my simulator. might just be because it is a simulator. will have to double-check this when deploying to prod.

Reply:
I think it's because it's a simulator. I took this link and copied it from my phone. I think it should work.